packages/shared-types/src/types.ts

```diff
@@ -1,4 +1,4 @@
-export interface IUser {
+export interface name {
   username?: string;
```

Comment:
I mean to change username  => name 😂

Reply:
bruh i asked you this and you said Yes in discord 💀